### DISCUSSION TITLE
Welcome to TTT-Bots-2 Discussions

### DISCUSSION THREAD

**Discussion by thebigsleepjoe** [thebigsleepjoe/TTT-Bots-2#4] [Announcements]
## 👋 Welcome!
If you have something to say that isn't a bug report or feature request, please leave it in this discussion section. Here are some things to talk about:
* Balancing
* Ideas for bot dialogs/chats
* Anything else that is related

Edit: I'd prefer if you created a new discussion over replying here! Though I will respond either way. Please consider creating an [Issue](https://github.com/thebigsleepjoe/TTT-Bots-2/issues) for suggestions/bugs.

**Please leave bot name suggestions in #47**

**Comment by GHDavor** [comment]
hey, i have an idea. is there a way you can make the bots join spectator deathmatch when they die? that would make the game less boring when you die

**Reply by thebigsleepjoe** [reply]
> hey, i have an idea. is there a way you can make the bots join spectator deathmatch when they die? that would make the game less boring when you die

I apologize for being late to your request. I don't frequently check the discussions -- I would recommend the Issues page for something like this anywho.

Unfortunately, the feature you are requesting is beyond the scope of this project. Implementing it would require restructuring the codebase, which would only benefit a small number of players compared to the workload. You are always welcome to submit a PR!

**Comment by SvveetMavis** [comment]
is there a way to disbale the message saying a map doesnt have a navmesh at the start of rounds

**Comment by thebigsleepjoe** [comment]
Svvveet, there isn’t currently a way to do this beyond a) generating a
navmesh, or b) creating a dummy (single square) navmesh. It shouldn’t be
notifying you every round; I’ll submit this as a bug report and look into
it when I have a moment.
